sdk/resourcemanager/azure-resourcemanager-compute/src/test/java/com/azure/resourcemanager/compute/TestInterfaceInheritance.java

```diff
@@ -0,0 +1,61 @@
+package com.azure.resourcemanager.compute;
+
+/**
+ * @author xiaofeicao
+ * @createdAt 2021-11-30 14:29
+ */
+public class TestInterfaceInheritance {
```

Comment:
where is this used?

Reply:
It's just a simple unit test for my knowledge on interface inheritance... Will delete it afterwards.